ibm_catalog.json

```diff
@@ -73,11 +76,11 @@
             "diagrams": [
               {
                 "diagram": {
-                  "caption": "Watson Discovery on IBM Cloud",
+                  "caption": "IBM Watson Discovery architecture",
                   "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-watsonx-discovery/main/reference-architecture/deployable-architecture-watson-discovery.svg",
                   "type": "image/svg+xml"
                 },
-                "description": "This architecture supports creating and configuring an instance of Watson Discovery."
+                "description": "This solution provisions an IBM Watson Discovery instance on IBM Cloud. Once the instance is successfully created, you will receive access to the Discovery dashboard, where you can begin connecting data sources and building AI-powered search projects. Optionally, you can integrate IBM Cloud Observability tools—such as logging, monitoring, and activity tracking event routing — to gain deeper operational insights and manage your Discovery workflows effectively."
```

Comment:
you are mentioning `IBM Cloud Observability tools` - tools can be replaced by services.
As you are talking about services then use service names. It needs to modify.